ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/EmailIntegrationForm.tsx

```diff
@@ -11,6 +20,7 @@ import FormMessage from 'Components/PatternFly/FormMessage';
 import FormCancelButton from 'Components/PatternFly/FormCancelButton';
 import FormTestButton from 'Components/PatternFly/FormTestButton';
 import FormSaveButton from 'Components/PatternFly/FormSaveButton';
+import ToggledPopoverElement from 'Components/ToggledPopoverElement';
```

Comment:
Not a blocker to merge: To reduce the effort to distinguish the many classic StackRox theme components from the fewer and newer components that are intended for PatternFly pages, we create them in PatternFly subfolder as in the `import` statements above.

Reply:
Great info, I'll go ahead and get that moved over